webapp/channels/src/components/advanced_text_editor/advanced_text_editor.tsx

```diff
@@ -332,6 +334,19 @@ const AdvancedTextEditor = ({
         isInEditMode,
     );
 
+    const onSubmit = useCallback((submittingDraft?: PostDraft, schedulingInfo?: SchedulingInfo, options?: CreatePostOptions) => {
```

Comment:
can you rename this to something like `handleSubmitWithErrorHandling`? Because we already have `handleSubmit` and `handleSubmitWrapper`, which is already a bit confusing, and now we have `onSubmit` as well, it will definitely create some confusion in the future.